website/project/model.py

```diff
@@ -293,6 +293,8 @@ class NodeLog(StoredObject):
     params = fields.DictionaryField()
     should_hide = fields.BooleanField(default=False)
 
+    was_connected_to = fields.ForeignField('node', list=True, backref='connected')
```

Comment:
Is this backref necessary? It adds bulk to the nodes it was attached to, and I don't see a reason to keep it.